packages/content/test/attributes/Website.test.ts

```diff
@@ -0,0 +1,105 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { Website } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute value type Website", function () {
```

Comment:
Most of this file consists of repetitive boilerplate, very low information density here.

Here's a suggestion: Create two separate test suites (`describe` blocks) in this file, one for valid and one for invalid websites. In each you define an array of URLs, which you test with a [parameterized jest test](https://medium.com/@Klaymant/parameterized-tests-with-typescript-jest-19f45a4a2925).

```ts
decribe("Test valid URLs", () => {
    const validUrls = [
        "www.google.com",
        "https://döner.cooking",
        // add all the weird URLs you want to test here.        
    ]
    
    test.each(validUrls)("url %s is recognized as valid", (url) => {
        const validWebsite = Website.from({value: url});
        expect(validWebsite.value.toString())toBe(url)
    })
})

// Same for invalid URLs
// ...
```

`test.each([...])` is basically equivalent to a for-loop running many tests with the added benefit that all the test results appear in the output individually, as if you had written them manually.